getting-started-guides/on-prem-cluster/standalone-python.md

```diff
@@ -85,25 +84,16 @@ Note that in this example the Master and Worker processes are both running on th
 
 Launch Mortgage ETL Example
 ---------------------------
-Variables required to run spark-submit command:
-```
-# path to xgboost4j_spark/libs
-export LIBS_PATH=/home/xgboost4j_spark/lib
-
-# Add sample.zip to py-files config
-export SPARK_PY_FILES=${LIBS_PATH}/samples.zip
-
-```
 
 Run spark-submit
-```
+``` bash
 ${SPARK_HOME}/bin/spark-submit \
     --master spark://$HOSTNAME:7077 \
     --executor-memory 32G \
     --conf spark.executor.resource.gpu.amount=1 \
     --conf spark.task.resource.gpu.amount=1 \
     --conf spark.plugins=com.nvidia.spark.SQLPlugin \
-    --py-files ${SPARK_PY_FILES} \
+    --py-files samples.zip \
```

Comment:
Why removing  ${LIBS_PATH}/samples.zip? Can samples.zip be found in current directory?